server/controllers/bankAccount.controllers.js

```diff
@@ -0,0 +1,57 @@
+import {
+  addAccount,
+  editAccount,
+  getAllAccounts,
+  approveAccount,
+  deleteAccount,
+} from '../services/bankAccount.service';
+import httpStatus from '../helpers/httpStatus';
+
+const BankAccountController = {
+  addAccount(req, res, next) {
+    const accountInfo = req.locals;
+    addAccount({ ...accountInfo, addedBy: req.user._id })
+      .then((account) => {
+        res.status(httpStatus.CREATED).json({ success: true, message: 'Account added', account });
```

Comment:
As explained above, lets use bankAccount` instead of `account`. Lets update this for other endpoints too.
```suggestion
        res.status(httpStatus.CREATED).json({ success: true, message: 'Account added', bankAccount });
```